src/app/global.css

```diff
@@ -1,3 +1,9 @@
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
```

Comment:
@pythonistical just wanted to note that you actually can just include this inside of the index.tsx file instead of here. I will include this in some of my cleaning on a branch coming soon. heres what that looks like: 

![image](https://github.com/pieces-app/example-ts/assets/55861512/2337b569-da22-44ec-9216-669cbc976164)